forte/data/data_store.py

```diff
@@ -1362,15 +1477,18 @@ def within_range(
         all_types = list(all_types)
         all_types.sort()
         if self._is_annotation(type_name):
-            if range_annotation is None:
+            if range_span is None:
                 yield from self.co_iterator_annotation_like(all_types)
             else:
-                for entry in self.co_iterator_annotation_like(all_types):
-                    if within_range(entry, range_annotation):
-                        yield entry
+                for entry in self.co_iterator_annotation_like(
+                    all_types,
+                    range_begin=range_span[constants.BEGIN_INDEX],
```

Comment:
Maybe it's better to keep the interface consistent? We can use either `range_begin, range_end` or `range_span` but maybe not both.